lib/common.js

```diff
@@ -285,7 +286,7 @@ exports.createTableForEnvDataSourcesAuditLogs = function(dataSource, options) {
     var errorMsg = currentStatus.error && currentStatus.error.userDetail ? currentStatus.error.userDetail : "NONE";
     var dataLength = dsAuditLogEntry.data ? dsAuditLogEntry.data.length : 0;
     var dataHash = dsAuditLogEntry.dataHash || "NONE";
-    table.push([dsAuditLogEntry.service.guid, dsAuditLogEntry.service.title, dsAuditLogEntry.endpoint, currentStatus.status, errorMsg, dataLength, dataHash, moment(dsAuditLogEntry.updateTimestamp).fromNow()]);
+    table.push([dsAuditLogEntry.service.guid, dsAuditLogEntry.service.title, dsAuditLogEntry.endpoint, currentStatus.status, errorMsg, dataLength, dataHash, moment(new Date(dsAuditLogEntry.updateTimestamp), DT_FORMAT).fromNow()]);
```

Comment:
how come new Date() is now required here but not in other places?

Reply:
@davidffrench, in this case, is because of the type of value returned by the backend service the, in this case, is required to convert to Date. You can check it here; https://github.com/feedhenry/fh-fhc/blob/master/lib/cmd/fh3/appforms/environments/data-sources/audit-logs.js#L29